tests-upgrade/AutoRestUpgradeTest.ps1

```diff
@@ -291,7 +290,7 @@ else
 {
     foreach($fileDetail in $fileList)
     {
-        if($fileDetail.Mode -eq 'd----' -and (!$fileDetail.Name.Startswith('Compare')))
+        if($fileDetail.Mode.Startswith('d') -and (!$fileDetail.Name.Startswith('Compare')))
```

Comment:
Please use file method instead of comparing mode string directly.

Reply:
Done,thanks